### DISCUSSION TITLE
v7.5.0

### DISCUSSION THREAD

**Discussion by antonmedv** [deployphp/deployer#3924] [General]
**The PHP deployment tool with support for popular frameworks out of the box.**

## What's Changed
* Fixed and improved **provision** recipe. 
  * Now with support for Ubuntu 20, 22, 24 and older!
  * New way of installing Node.
* New distribution and release process.
  * Plus, Docker image: [deployphp/deployer](https://hub.docker.com/r/deployphp/deployer)
  * Plus, Phive distribution
     ```
     phive install deployer
     ```

This is the last release of Deployer with PHP v7 support. Next version of Deployer v8 will support only PHP v8.

<hr /><em>This discussion was created from the release <a href='https://github.com/deployphp/deployer/releases/tag/v7.5.0'>v7.5.0</a>.</em>

**Comment by keroth-** [comment]
Hi!

Thanks for your release.
I had the same error in v7.4 but I thought 7.5 will correct this.

With the .gitlab-ci.yml example on deployer.org, I have this error:
`dep: command not found`

Like the alias does not exist in your image.
I tried with deploy too, same.

I'm sure there is a command running well but it is not the command describe on deployer.org website.

Maybe I do something wrong because I'm quite new in the CI/CD world ;)

Thanks,
Brice

**Reply by mrhackcz** [reply]
Hi!
Yeah, you are right. The problem is that no `dep` binary/alias exists. The Docker entrypoint is set to `php /bin/deployer.phar`. So the correct way to use the prebuild docker image is:

```
test-job:
  image:
    name: deployphp/deployer:v7
    entrypoint: [""]
  script:
    - php /bin/deployer.phar deploy
```

I think it would be better to have the `dep` command available.
```
diff --git a/Dockerfile b/Dockerfile
index f4c3c124..803f2f06 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -2,8 +2,8 @@ FROM php:8.3-cli-alpine
 
 RUN apk add --no-cache bash git openssh-client rsync
 
-COPY deployer.phar /bin/deployer.phar
+COPY --chmod 755 deployer.phar /bin/dep
 
 WORKDIR /app
 
-ENTRYPOINT ["php", "/bin/deployer.phar"]
+ENTRYPOINT ["/bin/dep"]
```

@antonmedv what about this change? Thanks for your work 🙏
And there is missing `v` in image name docs https://github.com/deployphp/deployer/blob/master/docs/ci-cd.md?plain=1#L64

**Reply by antonmedv** [reply]
Nice idea! Could you please write PR with update to docker image and docs? Thanks.

**Reply by antonmedv** [reply]
Note what this should be done on v7 branch for now.
